src/app/shared/entry/base-url.pipe.ts

```diff
@@ -4,21 +4,31 @@ import { Pipe, PipeTransform } from '@angular/core';
   name: 'baseUrl',
 })
 export class BaseUrlPipe implements PipeTransform {
+  private markdownBasePath: string;
```

Comment:
I don't think this needs to be a private field because it's only used inside `transform`. It can be a variable inside that function instead